File: ASP.NET/Epione.Domain/Entities/JObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Epione.Domain.Entities
{
    public class JObject
    {
        public string key { get; set; }
        public string value { get; set; }
        public JObject(string key, string value)
        {
            this.key = key;
            this.value = value;
        }
    }
}

File: Angular/src/app/model/jobject.ts
export class JObject {
  private _key: string;
  private _value: string;

  get key(): string {
    return this._key;
  }

  set key(value: string) {
    this._key = value;
  }

  get value(): string {
    return this._value;
  }

  set value(value: string) {
    this._value = value;
  }
}
